Sources/UXKit/AppKit/UXTableView-AppKit.swift

```diff
@@ -84,25 +116,35 @@
       reloadData(forRowIndexes: IndexSet.setForRowsInPathes(indexes),
                  columnIndexes: IndexSet(integer: 0))
     }
+    
+    func numberOfRows(inSection: Int) -> Int {
```

Comment:
macOS actually does support sections (group rows). Not sure whether/how this can be hooked up here. Maybe we should at least leave a comment that there is an actual peer in macOS.